test/views/coronavirus_local_restrictions/results_html_test.rb

```diff
@@ -0,0 +1,169 @@
+require "test_helper"
+require "gds_api/test_helpers/mapit"
+require_relative "../../support/coronavirus_local_restrictions_helpers"
+
+class ResultsHtmlTest < ActionView::TestCase
+  include GdsApi::TestHelpers::Mapit
+  include CoronavirusLocalRestrictionsHelpers
+  helper Rails.application.helpers
+
+  describe "current restrictions" do
+    test "rendering tier 4 results for a postcode in tier 4" do
+      render_tier_results(current_alert_level: 4)
+
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_pretext")
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_tier_label")
+      assert_includes rendered, area
```

Comment:
I don't think we need to assert on area here. I find it a bit of a surprising guest in the code as the variable isn't mentioned until this assertion.